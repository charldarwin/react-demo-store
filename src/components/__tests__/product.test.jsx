import React from "react"
import renderer from "react-test-renderer"
import Product from "../product"

describe("Testing Product component", () => {
    it("Renders", () => {
        const productItem = {
            "title": "Panes criollo",
            "imageSrc": "producto-1.jpg",
            "price": "44.00",
            "id": "1",
            "description": "Lorem ipsum description"
        }
        const tree = renderer
            .create(<Product data={productItem} />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})