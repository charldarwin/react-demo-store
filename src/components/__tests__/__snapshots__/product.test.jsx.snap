// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product component Renders 1`] = `
<div
  className="sc-AxhCb jXRveB"
>
  <div
    className="sc-AxmLO kMpVNt"
  />
  <h4
    className="sc-AxhUy gNPlep"
  >
    Panes criollo
  </h4>
  <div
    className="sc-AxgMl eHuohS"
  >
    Lorem ipsum description
  </div>
  <div
    className="sc-AxheI foIRpa"
  >
    <span>
      $ 
      44.00
    </span>
    <span
      className="quantity"
    >
      Kg
    </span>
  </div>
  <button
    className="sc-Axmtr eIfzHA"
    data-id="1"
    data-price="44.00"
    onClick={[Function]}
  >
    <span
      className="icon"
    >
      <img
        alt="ícono carrito"
        src="test-file-stub"
      />
    </span>
    <span>
      Añadir
    </span>
  </button>
  
</div>
`;
